requirements.txt

```diff
@@ -0,0 +1,5 @@
+Django>=2.1.3,<2.2.0
```

Comment:
Better use exact versions instead of ranges (for all libraries in this file).

Ranges are meant for libraries to be more compatible. However, your production code should be in a determined state.